google/cloud/aiplatform/training_jobs.py

```diff
@@ -3073,6 +3171,36 @@ def _run(
                 Worker pools pecs required to run job.
             managed_model (gca_model.Model):
                 Model proto if this script produces a Managed Model.
+            model_id (str):
```

Comment:
I noticed duplicate doc strings throughout the files. For instance, the doc string for `is_default_version` parameter was duplicated 19 times. Would it be possible to simplify the doc strings & only have it show up at the top-level functions? eg. we can remove these doc strings from the `_run` method and only keep it in `run`.

Reply:
I'm not sure we can avoid this, sadly. All the run() and _run() methods in this class accept different args, so you can't easily do docstring inheritance or borrow docstrings from another method. I tried earlier.

Open to clever solutions if you have one.

Reply:
In that case I think we should leave it as is for now. I have some concerns regarding the current way that the code base is organized as a whole, given that we have files that are ~2k-3k lines long and a large percentage of those lines of code are doc strings. But this has more to do with the continuous and ever-evolving discussion regarding best practice in programming, which is out of scope for this pull request.